widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.java

```diff
@@ -168,9 +163,6 @@ public void onItemRangeInserted(int positionStart, int itemCount) {
         }
     };
 
-    private Handler mainHandler = null;
```

Comment:
Were these removed because we use Rx now?

Reply:
yup